skyplane/obj_store/object_store_interface.py

```diff
@@ -27,6 +27,9 @@ def path(self) -> str:
     def region_tag(self) -> str:
         raise NotImplementedError()
 
+    def activate_requester(self):
```

Comment:
`activate_requester ` => `set_requester_pays_mode(status: bool)` so we can disable it as well